tests/drivers/opamp/opamp_stm32/testcase.yaml

```
@@ -0,0 +1,146 @@
+# Copyright (c) 2025 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+common:
+  tags:
+    - drivers
+    - opamp
+    - stm32g4
```

There are no tags related to SoC series.
```suggestion
  platform_allow:
    - nucleo_g474re
 ```
We'll be able to extn to other platfoms late if needed.